### PR TITLE
Show chat conversation in markdown

### PR DESCRIPTION
**Pull request by jellydn**
# What

Small improvement for adding heading for each role (user vs copilot)

# Why

By presenting conversations in markdown, users can more easily follow the flow of dialogue.

## Demo

[Image from Gyazo](https://gyazo.com/7e2c67f9c6018fc243c7503e70cc1e8b)

### CODE REVIEW

File: rplugin/python3/plugin.py [hunk 1]
```diff
@@ -44,11 +44,19 @@ def copilotChat(self, args: list[str]):
             # Create a new scratch buffer to hold the chat
             self.nvim.command("enew")
             self.nvim.command("setlocal buftype=nofile bufhidden=hide noswapfile")
-            # Set filetype as markdown and wrap
+            # Set filetype as markdown and wrap with linebreaks
             self.nvim.command("setlocal filetype=markdown")
```
Comment (jellydn) [inline rplugin/python3/plugin.py:48]:
```suggestion
```

File: rplugin/python3/plugin.py [hunk 2]
```diff
@@ -44,11 +44,19 @@ def copilotChat(self, args: list[str]):
...
-            self.nvim.command("setlocal wrap")
+            self.nvim.command("setlocal wrap linebreak")
```
Comment (jellydn) [inline rplugin/python3/plugin.py:49]:
```suggestion
            self.nvim.command("setlocal filetype=markdown wrap linebreak")
```

File: rplugin/python3/plugin.py [hunk 3]
```diff
@@ -58,3 +65,12 @@ def copilotChat(self, args: list[str]):
...
                     self.nvim.command("normal o")
+
+        self.nvim.command("normal o")
+        self.nvim.current.line += ""
+        self.nvim.command("normal o")
+        self.nvim.current.line += "---"
+        self.nvim.command("normal o")
+        self.nvim.current.line += (
+            "If you want to chat again, run :CopilotChat <prompt>."
+        )
```
Comment (gptlang) [inline rplugin/python3/plugin.py:76]: I don't think this is necessary.

Comment (jellydn) [reply rplugin/python3/plugin.py:76]: No problem. I just closed this PR. I thought that would be helpful for end user when Copilot is done and remind about usage.

**Review by gptlang — APPROVED** [review]

**Comment by gptlang** [comment]
Thank you!

**Pull request merged**